desktop_version/src/DiscordNetwork.c

```diff
@@ -0,0 +1,181 @@
+#include "MakeAndPlay.h"
+
+#ifndef MAKEANDPLAY
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <SDL.h>
+
+#include "Vlogging.h"
+
+// Library includes
+#include "discord_game_sdk.h"
+
+#if defined(_WIN32)
+#define DISCORD_LIBRARY "discord_game_sdk.dll"
+#elif defined(__APPLE__)
+#define DISCORD_LIBRARY "libdiscord_game_sdk.dylib"
+#elif defined(__linux__) || defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__HAIKU__) || defined(__DragonFly__)
+#define DISCORD_LIBRARY "libdiscord_game_sdk.so"
+#else
+#error DISCORD_LIBRARY: Unrecognized platform!
+#endif
+
+
+#define DISCORD_CLIENT_ID 1315544357532729447
+
+// TO TERRY/FLIBIT: You can create your own Discord instance at the Discord Developer Portal. This ID belongs to me, so just be aware that if my account was to get hacked, VVVVVV RPC would too. Use your own!
```

Comment:
@TerryCavanagh, does this already exist...?

Reply:
For reference, the [Discord docs](https://discord.com/developers/docs/developer-tools/game-sdk#step-2-create-your-app) do say this:

> If you're integrating our SDK into an already-released game, there's a good chance that we may already have an application in our database for your game! Reach out to our [Dev Support](https://dis.gd/devsupport) to learn more

I wonder whether the reasons to use the ID they already have are good enough reasons to bother lol (maybe so it smoothly takes over the "Playing VVVVVV", "last played 2 days ago", etc that was already showing without rich presence?)

Reply:
@Daaaav @flibitijibibo I have sent a message to Discord Support to see if this is true for VVVVVV. Hopefully I get a response before the 15th anniversary of VVVVVV. (I'm hoping this will be released then)

Reply:
I'm sure an application already exists for VVVVVV, as it existed on the Discord Game Store back when that existed.